paasta_tools/cli/schemas/chronos_schema.json

```diff
@@ -107,6 +107,15 @@
                         "uniqueItems": true
                     }
                 ]
+            },
+            "cfs_period_us": {
+                "type": "integer",
+                "minimum": "1000",
```

Comment:
For clarity, can you specify the `exclusiveMinimum` boolean key for this field and all of the other new ones?